### DISCUSSION TITLE
List items with number as first character don't render properly

### DISCUSSION THREAD

**Discussion by highball-it** [requarks/wiki#3604] [Error / Bug Report]
### Actual behavior
List items are not aligned, see:
![grafik](https://user-images.githubusercontent.com/5033336/89451514-a1d0cc00-d75c-11ea-9005-0142886165e4.png)

### Expected behavior
List items should be aligned correctly, like the bold formatted items.

### Steps to reproduce the behavior
Add list item with number as first character.

**Comment by Tete1805** [comment]
Hey there.

This is because it is interpreted as an ordered list inside an unordered list.
However, wrapping the numbers in the bold markup prevents the ordered list from being recognized as such, this is a nice trick.

Why not remove the unordered part and just have an ordered list?
```
1. ABC
2. ABC ...
```

You could also use the same trick but with an unbreakable space like so:
```
- &nbsp;1. ABC
- &nbsp;2. ABC
```

**Comment by Tete1805** [comment]
Also, if this helps, I just noticed you could get around this with other shapes like:
```
- 1.0 ABC
- 1.1 ABC
```
or :
```
- 1 ABC
- 2 ABC
```
or :
```
- 1 . ABC
- 2 . ABC
```
...

**Comment by highball-it** [comment]
> 
> 
> Why not remove the unordered part and just have an ordered list?
> 
> ```
> 1. ABC
> 2. ABC ...
> ```

I was using it for listing special dates like this:
```
- 01. April: Something
- 24. December: Something else
```

The bold markup came in handy, because it is better readable this way.
